talk/morelanguage/stl.tex

```diff
@@ -199,10 +199,10 @@
           return value + m_inc;
         }
       };
-      std::vector<int> v;
-      v.push_back(5); v.push_back(3); ...
+      std::vector<int> v{1, 2, 3};
+      const auto inc = 42;
```

Comment:
Why not use 42 directly? I mean, you can merge as is, but I'm just curious.

Reply:
Because the corresponding lambda example also has a separate variable now to show a capture.